package.json

```diff
@@ -85,5 +96,9 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
+  },
+  "volta": {
```

Comment:
We're not using volta, could you please remove it? or is it automatic from `ember-cli-update`?

Reply:
@villander removed